integration-tests/src/test/java/oracle/kubernetes/operator/ItOperatorUpgrade.java

```diff
@@ -322,15 +272,11 @@ private void setupOperatorAndDomain(String operatorGitRelease, String operatorRe
     domNs.add(DOM_NS);
     operatorMap.put("domainNamespaces", domNs);
     operator = TestUtils.createOperator(operatorMap, Operator.RestCertType.LEGACY);
-    // TestUtils.exec("kubectl get all --all-namespaces", true);
-
-    // Map<String, Object> wlstDomainMap = TestUtils.loadYaml(DOMAININIMAGE_WLST_YAML);
-    Map<String, Object> wlstDomainMap = createDomainInImageMap(getNewSuffixCount(), false, testClassName);
+    Map<String, Object> wlstDomainMap = createDomainMap(getNewSuffixCount(),testClassName);
```

Comment:
Why are we switching from Domain in Image to Domain in PV?  If we need to test Domain in PV here, we should ADD it (or add a copy of this test for PV), not remove Domain in Image.

Reply:
The purpose of the Upgrade Usecase is to verify the ability of the Upgraded Operator to manage the existing Weblogic Domain. Will the upgrade process takes a different code path based on the domain image is generated.  

Another Reason :   I have modified the domain-in-image sample script  ( in develop) to skip the domain image creation if the image exits to reduce the test execution time.  But this change is not available in master ( 2.5.0 branch), so I can not use this  feature while creating domain in older release to start the Domain

Reply:
@ddsharpe does this resolve your concern?

Reply:
@ddsharpe

Reply:
confirmed derek is ok with the response